httpserver/src/main/kotlin/tbdex/sdk/httpserver/models/ExchangesApi.kt

```diff
@@ -1,65 +1,70 @@
  */
 interface ExchangesApi {
 
   /**
-   * Retrieves the exchange(s) with the specified ID(s).
+   * Retrieves the exchange with the specified ID.
    *
-   * @param id A list of exchange IDs to retrieve. If null, returns null.
-   * @return A list of [MessageKind] representing exchanges with the specified ID(s), or null if not found.
+   * @param id The ID of the exchange to retrieve.
+   * @return A list of [Message] representing the exchange with the specified ID.
    */
-  fun getExchange(id: List<String>? = null): List<MessageKind>?
+  fun getExchange(id: String): List<Message>
 
   /**
    * Retrieves a list of exchanges based on the provided filter.
    *
    * @param filter The filter criteria for retrieving exchanges. If null, returns all exchanges.
-   * @return A list of lists of [MessageKind] representing exchanges based on the filter, or null if none are found.
+   * @return A list of lists of [Message] representing exchanges based on the filter.
    */
-  fun getExchanges(filter: GetExchangesFilter? = null): List<List<MessageKind>>?
+  fun getExchanges(filter: GetExchangesFilter? = null): List<List<Message>>
 
   /**
-   * Retrieves the Request for Quote (RFQ) associated with the specified exchange.
+   * Retrieves the Request for Quote (RFQ) for the specified exchange ID.
    *
-   * @param exchangeId The ID of the exchange associated with the RFQ. If null, returns null.
-   * @return The [Rfq] associated with the specified exchange, or null if not found.
+   * @param exchangeId The ID of the exchange to retrieve the RFQ for.
+   * @return The [Rfq] object representing the RFQ for the specified exchange.
    */
-  fun getRfq(exchangeId: String? = null): Rfq?
+  fun getRfq(exchangeId: String): Rfq
 
   /**
-   * Retrieves the Quote associated with the specified exchange.
+   * Retrieves the Quote for the specified exchange ID.
    *
-   * @param exchangeId The ID of the exchange associated with the Quote. If null, returns null.
-   * @return The [Quote] associated with the specified exchange, or null if not found.
+   * @param exchangeId The ID of the exchange to retrieve the Quote for.
+   * @return The [Quote] object representing the Quote for the specified exchange.
    */
-  fun getQuote(exchangeId: String? = null): Quote?
+  fun getQuote(exchangeId: String): Quote
 
   /**
-   * Retrieves the Order associated with the specified exchange.
+   * Retrieves the Order for the specified exchange ID.
    *
-   * @param exchangeId The ID of the exchange associated with the Order. If null, returns null.
-   * @return The [Order] associated with the specified exchange, or null if not found.
+   * @param exchangeId The ID of the exchange to retrieve the Order for.
+   * @return The [Order] object representing the Order for the specified exchange.
    */
-  fun getOrder(exchangeId: String? = null): Order?
+  fun getOrder(exchangeId: String): Order
```

Comment:
Ditto:
I think you're intending this throws if none is found. We should document.